src/bin/cli/rewards/voters.rs

```diff
@@ -22,12 +23,18 @@ pub struct VotersRewards {
     /// Reward (in LOVELACE) to be distributed
     #[structopt(long = "total-rewards")]
     total_rewards: u64,
+
+    #[structopt(long = "votes-count-path")]
```

Comment:
Nitpick: I think we don't need this if we already have `#[structopt(rename_all = "kebab-case")]`. Don't know why we've been doing this tbh.

Reply:
By using just the `long` attribute force you to use it as a flag parameter instead of as a positional parameter. Imho it is clearer (although I understand it is more bothersome to write the full flag) that just positional parameters.

Reply:
I think he meant you could simply write `#[structopt(long)]`

Reply:
Aah, well, yes. Don't know what was I thinking about though.